libs/test-utils/src/expect_print.ts

```diff
@@ -0,0 +1,138 @@
+import { render, RenderResult } from '@testing-library/react';
+import { Optional, Printer, PrintOptions } from '@votingworks/types';
+import { advanceTimersAndPromises } from './advance_timers';
+import { assert } from './assert';
+
+export const fakePrintElement = jest.fn(
+  (_element: JSX.Element, printer: Printer, printOptions: PrintOptions) => {
+    return printer.print(printOptions);
+  }
+);
+
+export const fakePrintElementWhenReady = jest.fn(
+  async (
+    renderElement: (onReadyToPrint: () => void) => JSX.Element,
+    printer: Printer,
+    printOptions: PrintOptions
+  ) => {
+    return printer.print(printOptions);
```

Comment:
We never actually end up using the mock implementations, just the signatures and the mock object, but I have it calling print in case that is ever expected.